core/templates/base-components/feedback-modal.component.html

```
@@ -181,7 +181,9 @@ <h3>{{ title | translate }}</h3>
         | translate
         }}
       </button>
-      <button class="btn btn-success" (click)="submit()">
+      <button class="btn btn-success" 
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)